### ISSUE TITLE
Change "Subscribe" Semantics for Blocking/Muting Lists

### ISSUE THREAD

**Issue by zanfranceschi** [bluesky-social/social-app#5269]
**Is your feature request related to a problem? Please describe.**
In the moderation lists context, me and some other folks find it's very confusing to "[Subscribe](https://github.com/bluesky-social/social-app/blob/fc25992070633af9c242712fe6234a518200ef9b/src/view/screens/ProfileList.tsx#L700)" to a list of accounts one wants to block or mute. Maybe the word "Subscribe" generally means you want to subscribe to something you want to consume and not block/mute.

**Describe the solution you'd like**
Change the "Subscribe" semantics to something more related to moderation – examples: "moderation actions", "moderation", "action", "block or mute", etc

![image](https://github.com/user-attachments/assets/dfe42101-887c-4d28-8de8-c5b69ae94abf)

**Comment by DanielHe4rt** [comment]
Agree. First time using the "block all" list, I tried everything before click in "Subscribe" since it was an toxic account list.

Good catch @zanfranceschi!

**Comment by LucasMorais1998** [comment]
I admit that the "Subscribe" also confused me a bit at first, because if it's a list of people I don't want to be in contact with, "Subscribe" kind of makes us subconsciously think of the opposite.

**Comment by scottdotweb** [comment]
"Subscribe" is basically too low level. If a list has "block all" and "mute all" buttons, those imply a subscription to the list to apply the action to its members, so there's no need to have it as an action first.

**Comment by jamesonwilliams** [comment]
I just closed similar https://github.com/bluesky-social/social-app/issues/6167 - will throw my 2 cents into the pool after thinking about it a bit more:

There are multiple valuable actions one might take on a mod list: block, mute, subscribe to all. The bug here is that we have this subscribe button acting as a menu, when all three options should be presented as primary actions in the main presentation on this screen, as a header element under the mod list name/description.

**Comment by zanfranceschi** [comment]
closing it – thanks

**Comment by scottdotweb** [comment]
This isn't completed, though?

**Comment by jamesonwilliams** [comment]
@zanfranceschi to be clear, I closed the other one as a duplicate of this open and valid issue
